index.html

```diff
@@ -1,29 +1,17 @@
 <!DOCTYPE html>
```

Comment:
Why did you not create another html file? We still need the login file. You should create another html file and add the markdown to that.

Reply:
I thought I can just replace the content of the pulled html file.

I will close this PR and I will create another one to make everything clear